app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
> Building up an HTML form, validating, and joining parameters based on question type (checkboxes or radio buttons) feel like enough similarities to justify a shared implementation.

I think I would agree if it weren't for the way service search treats the lot as a mode-switch parameter, which adds a lot of special-case-ness to that code, which I thought interfering with was beyond the 2-point, just-get-this-out nature of this story.

I actually didn't realize that we only currently use wtforms for login forms - thought it was used for some of the admin forms too, but no.